docs/spec_draft.md

```diff
@@ -229,6 +229,8 @@ the IEEE-754 specification.
 // %result: [[6, 8], [10, 12]]
 ```
 
+&nbsp;[More Examples](https://github.com/sdasgup3/stablehlo/blob/main/stablehlo/tests/interpret_add.mlir)
```

Comment:
Looks like this link is pointing to your fork of the repo :) How about using a relative link instead? I think `stablehlo/tests/interpret_add.mlir` will work correctly.